db/schema.rb

```diff
@@ -212,11 +212,11 @@
   create_table "web_of_science_source_records", force: :cascade do |t|
     t.boolean  "active"
     t.string   "database",           limit: 255
-    t.text     "source_data",        limit: 65535
+    t.text     "source_data",        limit: 16777215
     t.string   "source_fingerprint", limit: 255
     t.string   "uid",                limit: 255
-    t.datetime "created_at",                       null: false
```

Comment:
These 2 lines shouldn't have been altered, right?  Or am I missing something?

Reply:
I also looked carefully at that.  It's the auto-format.  It's padding out the `null` column now because now the `limit` column is wider.  It's a no-op, just a format change.  If we don't accept it here, it will just keep happening every time rails auto-formats this file anyway.